### ISSUE TITLE
Cert not created when --home and --cert-home are defined during install.

### ISSUE THREAD

**Issue by republicus** [Neilpang/acme.sh#2084]
```
./acme.sh` -v
https://github.com/Neilpang/acme.sh
v2.8.1
```

Certificates are not created when `--home` and `--cert-home` are defined during install.
Issuing and renewing certificates **report success** but no certs are created or updated.

I initially was running acme.sh from `/root` and certs were being created in the default `/root/.acme/`
After an install outside of `/root` no certificates are created.

Permissions are wide open. I have set chmod 777 to the path defined in `--cert-home` 
No joy. No errors are reported. On the contrary, acme.sh reports success and correctly identifies the path but never creates the cert folder or files.

Steps to reproduce
------------------
```
./acme.sh --install --home /shares/letsencrypt/acme --config-home /shares/letsencrypt/acme --cert-home /shares/letsencrypt/certs

./acme.sh --home /shares/letsencrypt/acme --[issue/renew]
```

Results are identical when issued or renewed, forced or not, ecc or standard.
No certs are created even though it is reported successful.

```
./acme.sh --home /shares/letsencrypt/acme --issue --apache -d mydomain.net -d www.mydomain.net -d mail.mydomain.net -d ftp.mydomain.net --keylength ec-256
[Wed Feb  6 21:33:24 UTC 2019] Checking if there is an error in the apache config file before starting.
[Wed Feb  6 21:33:24 UTC 2019] OK
[Wed Feb  6 21:33:24 UTC 2019] JFYI, Config file /etc/httpd/conf/httpd.conf is backuped to /shares/letsencrypt/acme/httpd.conf
[Wed Feb  6 21:33:24 UTC 2019] In case there is an error that can not be restored automatically, you may try restore it yourself.
[Wed Feb  6 21:33:24 UTC 2019] The backup file will be deleted on success, just forget it.
[Wed Feb  6 21:33:25 UTC 2019] Creating domain key
[Wed Feb  6 21:33:25 UTC 2019] The domain key is here: /share/letsencrypt/certs/mydomain.net_ecc/mydomain.net.key
[Wed Feb  6 21:33:25 UTC 2019] Multi domain='DNS:mydomain.net,DNS:www.mydomain.net,DNS:mail.mydomain.net,DNS:ftp.mydomain.net'
[Wed Feb  6 21:33:25 UTC 2019] Getting domain auth token for each domain
[Wed Feb  6 21:33:25 UTC 2019] Getting webroot for domain='mydomain.net'
[Wed Feb  6 21:33:25 UTC 2019] Getting new-authz for domain='mydomain.net'
[Wed Feb  6 21:33:26 UTC 2019] The new-authz request is ok.
[Wed Feb  6 21:33:26 UTC 2019] Getting webroot for domain='www.mydomain.net'
[Wed Feb  6 21:33:26 UTC 2019] Getting new-authz for domain='www.mydomain.net'
[Wed Feb  6 21:33:26 UTC 2019] The new-authz request is ok.
[Wed Feb  6 21:33:26 UTC 2019] Getting webroot for domain='mail.mydomain.net'
[Wed Feb  6 21:33:26 UTC 2019] Getting new-authz for domain='mail.mydomain.net'
[Wed Feb  6 21:33:27 UTC 2019] The new-authz request is ok.
[Wed Feb  6 21:33:27 UTC 2019] Getting webroot for domain='ftp.mydomain.net'
[Wed Feb  6 21:33:27 UTC 2019] Getting new-authz for domain='ftp.mydomain.net'
[Wed Feb  6 21:33:27 UTC 2019] The new-authz request is ok.
[Wed Feb  6 21:33:27 UTC 2019] Verifying: mydomain.net
[Wed Feb  6 21:33:30 UTC 2019] Success
[Wed Feb  6 21:33:30 UTC 2019] Verifying: www.mydomain.net
[Wed Feb  6 21:33:33 UTC 2019] Success
[Wed Feb  6 21:33:33 UTC 2019] Verifying: mail.mydomain.net
[Wed Feb  6 21:33:35 UTC 2019] Success
[Wed Feb  6 21:33:35 UTC 2019] Verifying: ftp.mydomain.net
[Wed Feb  6 21:33:38 UTC 2019] Success
[Wed Feb  6 21:33:39 UTC 2019] Verify finished, start to sign.
[Wed Feb  6 21:33:40 UTC 2019] Cert success.
-----BEGIN CERTIFICATE-----
[...]
-----END CERTIFICATE-----
[Wed Feb  6 21:33:40 UTC 2019] Your cert is in  /share/letsencrypt/certs/mydomain.net_ecc/mydomain.net.cer
[Wed Feb  6 21:33:40 UTC 2019] Your cert key is in  /share/letsencrypt/certs/mydomain.net_ecc/mydomain.net.key
[Wed Feb  6 21:33:40 UTC 2019] The intermediate CA cert is in  /share/letsencrypt/certs/mydomian.net_ecc/ca.cer
[Wed Feb  6 21:33:40 UTC 2019] And the full chain certs is there:  /share/letsencrypt/certs/mydomain.net_ecc/fullchain.cer
```

Empty `/share/letsencrypt/certs/` after "success"

**Comment by republicus** [comment]
Belay my last. It appears I made a mistake in my install script. Pointing to `/share/[...]` instead of `/shares/[...]`

Though for clarity: Does the installer have a preference for the parameter `--cert-home` or `--certhome` ? The [How to Install wiki](https://github.com/Neilpang/acme.sh/wiki/How-to-install) calls for `--cert-home`; while the [Options and Params wiki](https://github.com/Neilpang/acme.sh/wiki/Options-and-Params) 
 instead calls for `--certhome` parameters.

This ticket can be closed less it be transformed into a request to address the documentation on install parameters.

**Comment by kacpernowak1981** [comment]
Hello
I have the same issue. 

Acme installation
`./acme.sh --install --home /root/acme-letsencrypt --cert-home /s3_bucket/crt/ --config-home /root/acme-letsencrypt --accountemail "sysadmin@mydomain"`

Output
`[Wed Nov  4 14:19:14 UTC 2020] LE_WORKING_DIR='/root/acme-letsencrypt'
https://github.com/acmesh-official/acme.sh
v2.8.8
[Wed Nov  4 14:19:14 UTC 2020] Running cmd: install
[Wed Nov  4 14:19:14 UTC 2020] readlink exists=0
[Wed Nov  4 14:19:14 UTC 2020] dirname exists=0
[Wed Nov  4 14:19:14 UTC 2020] Lets find script dir.
[Wed Nov  4 14:19:14 UTC 2020] _SCRIPT_='./acme.sh'
[Wed Nov  4 14:19:14 UTC 2020] _script='/root/acme.sh/acme.sh'
[Wed Nov  4 14:19:14 UTC 2020] _script_home='/root/acme.sh'
[Wed Nov  4 14:19:14 UTC 2020] Using config home:/root/acme-letsencrypt
[Wed Nov  4 14:19:14 UTC 2020] ACCOUNT_CONF_PATH='/root/acme-letsencrypt/account.conf'
[Wed Nov  4 14:19:14 UTC 2020] config file is empty, can not read DEFAULT_ACME_SERVER
[Wed Nov  4 14:19:14 UTC 2020] default_acme_server
[Wed Nov  4 14:19:14 UTC 2020] ACME_DIRECTORY='https://acme-v02.api.letsencrypt.org/directory'
[Wed Nov  4 14:19:14 UTC 2020] _ACME_SERVER_HOST='acme-v02.api.letsencrypt.org'
[Wed Nov  4 14:19:14 UTC 2020] CA_CONF='/root/acme-letsencrypt/ca/acme-v02.api.letsencrypt.org/ca.conf'
[Wed Nov  4 14:19:14 UTC 2020] curl exists=0
[Wed Nov  4 14:19:14 UTC 2020] crontab exists=0
[Wed Nov  4 14:19:14 UTC 2020] openssl exists=0
[Wed Nov  4 14:19:14 UTC 2020] socat exists=0
[Wed Nov  4 14:19:14 UTC 2020] Installing to /root/acme-letsencrypt
[Wed Nov  4 14:19:14 UTC 2020] Installed to /root/acme-letsencrypt/acme.sh
[Wed Nov  4 14:19:14 UTC 2020] Using config home:/root/acme-letsencrypt
[Wed Nov  4 14:19:14 UTC 2020] ACCOUNT_CONF_PATH='/root/acme-letsencrypt/account.conf'
[Wed Nov  4 14:19:14 UTC 2020] ACME_DIRECTORY='https://acme-v02.api.letsencrypt.org/directory'
[Wed Nov  4 14:19:14 UTC 2020] _ACME_SERVER_HOST='acme-v02.api.letsencrypt.org'
[Wed Nov  4 14:19:14 UTC 2020] CA_CONF='/root/acme-letsencrypt/ca/acme-v02.api.letsencrypt.org/ca.conf'
[Wed Nov  4 14:19:14 UTC 2020] APP
[Wed Nov  4 14:19:14 UTC 2020] 1:export LE_WORKING_DIR="/root/acme-letsencrypt"
[Wed Nov  4 14:19:14 UTC 2020] APP
[Wed Nov  4 14:19:14 UTC 2020] 2:export LE_CONFIG_HOME="/root/acme-letsencrypt"
[Wed Nov  4 14:19:14 UTC 2020] APP
[Wed Nov  4 14:19:14 UTC 2020] 3:alias acme.sh="/root/acme-letsencrypt/acme.sh --config-home '/root/acme-letsencrypt'"
[Wed Nov  4 14:19:14 UTC 2020] Found profile: /root/.bashrc
[Wed Nov  4 14:19:14 UTC 2020] Installing alias to '/root/.bashrc'
[Wed Nov  4 14:19:14 UTC 2020] APP
[Wed Nov  4 14:19:14 UTC 2020] 104:. "/root/acme-letsencrypt/acme.sh.env"
[Wed Nov  4 14:19:14 UTC 2020] OK, Close and reopen your terminal to start using acme.sh
[Wed Nov  4 14:19:14 UTC 2020] Using config home:/root/acme-letsencrypt
[Wed Nov  4 14:19:14 UTC 2020] ACCOUNT_CONF_PATH='/root/acme-letsencrypt/account.conf'
[Wed Nov  4 14:19:14 UTC 2020] ACME_DIRECTORY='https://acme-v02.api.letsencrypt.org/directory'
[Wed Nov  4 14:19:14 UTC 2020] _ACME_SERVER_HOST='acme-v02.api.letsencrypt.org'
[Wed Nov  4 14:19:14 UTC 2020] CA_CONF='/root/acme-letsencrypt/ca/acme-v02.api.letsencrypt.org/ca.conf'
[Wed Nov  4 14:19:14 UTC 2020] APP
[Wed Nov  4 14:19:14 UTC 2020] 11:CERT_HOME='/s3_sj3-staging-bucket-certificates/crt'
[Wed Nov  4 14:19:14 UTC 2020] Using config home:/root/acme-letsencrypt
[Wed Nov  4 14:19:14 UTC 2020] ACCOUNT_CONF_PATH='/root/acme-letsencrypt/account.conf'
[Wed Nov  4 14:19:14 UTC 2020] ACME_DIRECTORY='https://acme-v02.api.letsencrypt.org/directory'
[Wed Nov  4 14:19:14 UTC 2020] _ACME_SERVER_HOST='acme-v02.api.letsencrypt.org'
[Wed Nov  4 14:19:14 UTC 2020] CA_CONF='/root/acme-letsencrypt/ca/acme-v02.api.letsencrypt.org/ca.conf'
[Wed Nov  4 14:19:14 UTC 2020] crontab exists=0
[Wed Nov  4 14:19:14 UTC 2020] crontab exists=0
[Wed Nov  4 14:19:14 UTC 2020] Installing cron job
[Wed Nov  4 14:19:14 UTC 2020] uname exists=0
[Wed Nov  4 14:19:14 UTC 2020] bash exists=0
[Wed Nov  4 14:19:14 UTC 2020] Good, bash is found, so change the shebang to use bash as preferred.
[Wed Nov  4 14:19:14 UTC 2020] OK`

Cert issue

`./acme.sh/acme.sh --issue --days 180 -d mydomain.com -w /var/www/wellknown --debug 3`

Output
`[Wed Nov  4 14:25:25 UTC 2020] Installing to /root/acme-letsencrypt
[Wed Nov  4 14:25:25 UTC 2020] Installed to /root/acme-letsencrypt/acme.sh
[Wed Nov  4 14:25:25 UTC 2020] Installing alias to '/root/.bashrc'
[Wed Nov  4 14:25:25 UTC 2020] OK, Close and reopen your terminal to start using acme.sh
[Wed Nov  4 14:25:25 UTC 2020] Installing cron job
14 0 * * * "/root/acme-letsencrypt"/acme.sh --cron --home "/root/acme-letsencrypt" --config-home "/root/acme-letsencrypt" > /dev/null
[Wed Nov  4 14:25:25 UTC 2020] Good, bash is found, so change the shebang to use bash as preferred.
[Wed Nov  4 14:25:26 UTC 2020] OK`

any ideas what is wrong? This functionality is very important.
Thanks

**Comment by kacpernowak1981** [comment]
> Belay my last. It appears I made a mistake in my install script. Pointing to `/share/[...]` instead of `/shares/[...]`
> 
> Though for clarity: Does the installer have a preference for the parameter `--cert-home` or `--certhome` ? The [How to Install wiki](https://github.com/Neilpang/acme.sh/wiki/How-to-install) calls for `--cert-home`; while the [Options and Params wiki](https://github.com/Neilpang/acme.sh/wiki/Options-and-Params)
> instead calls for `--certhome` parameters.
> 
> This ticket can be closed less it be transformed into a request to address the documentation on install parameters.

It's not important, in acme.sh code is some like this:
`--cert-home | --certhome`

**Comment by neilpang** [comment]
Closing as stale: no activity for over 18 months. If this still applies on the current release, please open a new issue with the `--debug 2` log.

**Comment by neilpang** [comment]
Not a bug: reporter confirmed a typo in his install script (/share vs /shares); --cert-home and --certhome are aliases.
